XlsxSerializerCom/Writer/BinaryReader.h

```diff
@@ -48,7 +48,7 @@
 
 #include "../../ASCOfficeDocxFile2/BinReader/DefaultThemeWriter.h"
 
-#include "../../../../ASCOfficePPTXFile/PPTXFormat/Theme.h"
+#include "../../ASCOfficePPTXFile/PPTXFormat/Theme.h"

```

Comment:
Thank you. Fixed at https://github.com/ONLYOFFICE/core/commit/e31f1010219feb87af1991238093071ec73a8fb5#diff-6e06c833d00c0120e5c85ffb4ab9b193L51